BasicClasses/BasicClasses.tex

```diff
@@ -463,7 +463,7 @@ \subsection{Number}
 1 + 2.5     --> 3.5             "Addition of two numbers"
 3.4 * 5      --> 17.0           "Multiplication of two numbers"
 8 / 2         --> 4                 "Division of two numbers"
-10 - 8.3   --> 1.7              "Subtraction of two numbers"
+10 - 8.5   --> 1.5              "Subtraction of two numbers"
```

Comment:
This one works around the actual output of the original input, `1.6999999999999993`. Should we explain this phenomenon at some other point?

Reply:
No, this is basic CS knowledge and not the focus of the book